docs/finetune/data

```diff
@@ -0,0 +1,85 @@
+timestamp,prompt,completion
```

Comment:
Please call this folder `data.csv` (not just `data`).

Also, please move this folder to `docs/use-cases/automated_finetuning` folder, and please update the link in `use-cases/automated_finetuning/openai.mdx` tutorial.